README.md

```diff
@@ -1,7 +1,6 @@
 ![logo](logo/fplus.png)
 
 [![CI](https://github.com/Dobiasd/FunctionalPlus/workflows/ci/badge.svg)](https://github.com/Dobiasd/FunctionalPlus/actions)
-[![Build Status Travis](https://travis-ci.org/Dobiasd/FunctionalPlus.svg?branch=master)][travis]
```

Comment:
The following line

`[travis]: https://travis-ci.org/Dobiasd/FunctionalPlus`

can also be removed completely from the README.

Reply:
Ah, thanks